.github/workflows/x-create-releases.yml

```diff
@@ -0,0 +1,38 @@
+name: X Create releases
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+      prerelease:
+        required: false
+        type: boolean
+        default: true
+    secrets:
+      GH_ORG:
+        required: true
+      GH_TOKEN:
+        required: true
+jobs:
+  create-releases:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    strategy:
+      matrix:
+        repository:
+          - keycloak
+          - keycloak-documentation
+          - keycloak-nodejs-admin-client
+    steps:
+      - name: Get releases
+        id: get-releases
+        run: echo ::set-output name=releases::$(gh release list --repo ${{ secrets.GH_ORG }}/${{ matrix.repository }})
+
+      - name: Delete existing release
+        if: contains(steps.get-releases.outputs.releases, inputs.tag)
+        run: gh release delete ${{ inputs.tag }} --repo ${{ secrets.GH_ORG }}/${{ matrix.repository }} --yes
+
+      - name: Create a release
+        run: gh release create ${{ inputs.tag }} --repo ${{ secrets.GH_ORG }}/${{ matrix.repository }} --title ${{ inputs.tag }} --draft ${{ inputs.prerelease && '--prerelease' || '' }}
```

Comment:
Not sure if this will send notifications for nightlies or not, but we can do it this way for now, then readjust back to keeping the nightly release and emptying it if we need to.

Reply:
Yeah, this is something we need to figure out.